packages/c/crun/package.yml

```diff
@@ -0,0 +1,27 @@
+name       : crun
+version    : 1.16.1
+release    : 1
+source     :
+    - https://github.com/containers/crun/releases/download/1.16.1/crun-1.16.1.tar.zst : 473968be42b35eaf9477a11855b1deaa3e1072b0604a5c20a470e2f108280afb
+homepage   : https://github.com/containers/crun
+license    : |
+    - GPL-2.0-only
+    - LGPL-2.1-only
+component  : virt
```

Comment:
Add `libsplit: no` to the package.yml file. This will move the `.so` file into the main package, there's no need to create a crun-devel package when nothing links against it.